Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:2.7.12
```

Comment:
Right now the app is written for Python 2.7. I think that rewriting in Python 3 may need to be done, but it is outside the scope of this pull request.